pkgs/applications/networking/instant-messengers/discord/linux.nix

```diff
@@ -62,6 +62,8 @@
   openasar,
   withVencord ? false,
   vencord,
+  withMoonlight ? false,
```

Comment:
I assume this is incompatible with Vencord. Could you add an assertion to the beginning of this derivation to catch cases where both are enabled?

i.e.:
```nix
assert lib.assertMsg (!(withMoonlight && withVencord)) "discord: Moonlight and Vencord can not be enabled at the same time";
```